benchmarks/encoder/benchmark

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+mkdir -p /artifacts/${RUN_ID}
+
+RESULTS_1=/artifacts/${RUN_ID}/${DDTRACE_GIT_COMMIT_ID_1}.json
+RESULTS_2=/artifacts/${RUN_ID}/${DDTRACE_GIT_COMMIT_ID_2}.json
+
+# append venvs with ddtrace to sys.path
+
+PYTHONPATH=/app/.venv_1/lib/python3.9/site-packages \
+    python scenario.py \
+    --copy-env \
+    --fast \
+    -o ${RESULTS_1}
+
+PYTHONPATH=/app/.venv_2/lib/python3.9/site-packages \
+    python scenario.py \
+    --copy-env \
+    --fast \
+    -o ${RESULTS_2}
```

Comment:
i'm personally ok with the duplication here for now but of course a third occurrence would worth DRY-ing

Reply:
yep, this is code I assume we'll replace rather than keep. repeating ourselves here is really just cause I didn't to spend more time on creating these virtual environments and running commands within them. this is probably where we'd instead use riot/tox/etc.